### PR TITLE
Tooltips and Overflow menus

### PR DESCRIPTION
**Pull request by AaronJThompson**
# Description

This PR adds tooltip and overflow menu components. These are used in RC3 to provide extra information and interactions to users. 

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## Change status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/Tooltip/index.js [hunk 1]
```diff
@@ -20,7 +20,12 @@ const Tooltip = props => {
     }
   })();
 
-  return <tooltip bgColor={bgColor} color={color} text={text} />;
+  return (
+    <TooltipWrapper>
+      {props.children}
```
Comment (ropeks) [inline src/components/Tooltip/index.js]: hey, what does this line do? @AaronJThompson

File: src/components/SideNav/index.js [hunk 2]
```diff
@@ -6,14 +6,17 @@ import bravoWhite from '../../assets/bravo-white.svg';
...
     <SideNavContainer>
-      <Link to="/">
-        <img src={bravoWhite} alt="Bravo" />
-      </Link>
+      <TooltipDark text="testing">
+        <Link to="/">
+          <img src={bravoWhite} alt="Bravo" />
+        </Link>
+      </TooltipDark>
       {/* { slack.isLoggedIn && <SideNavMenu /> } */}
```
Comment (ropeks) [inline src/components/SideNav/index.js]: should this remain commented out?

Comment (AaronJThompson) [reply src/components/SideNav/index.js]: This was removed in the latest commit and was only for testing

File: src/components/Tooltip/index.js (hunk 1 above)
Comment (AaronJThompson) [reply src/components/Tooltip/index.js]: Which line are you referring to?

**Pull request merged**